plugins/woocommerce/src/Blocks/BlockTypes/MiniCartTitleItemsCounterBlock.php

```diff
@@ -11,4 +13,68 @@ class MiniCartTitleItemsCounterBlock extends AbstractInnerBlock {
 	 * @var string
 	 */
 	protected $block_name = 'mini-cart-title-items-counter-block';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content    Block content.
+	 * @param WP_Block $block      Block instance.
+	 * @return string Rendered block type output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		if ( Features::is_enabled( 'experimental-iapi-mini-cart' ) ) {
+			return $this->render_experimental_iapi_title_label_block();
+		}
+		return $content;
+	}
+
+	/**
+	 * Render the interactivity API powered experimental title block.
+	 *
+	 * @return string Rendered block type output.
+	 */
+	protected function render_experimental_iapi_title_label_block() {
+		$cart            = $this->get_cart_instance();
+		$cart_item_count = $cart ? $cart->get_cart_contents_count() : 0;
+		// translators: %d number of items in the cart.
+		$cart_item_text = _n( '(%d item)', '(%d items)', $cart_item_count, 'woocommerce' );
+
+		// translators: item is an item in cart.
+		$singular = __( '(%d item)', 'woocommerce' );
+		// translators: items is items in a cart.
+		$plural = __( '(%d items)', 'woocommerce' );
+
+		$context = array(
+			'singularItemsText' => $singular,
+			'pluralItemsText'   => $plural,
+		);
```

Comment:
As these don't change (do they?), maybe we try using `wp_interactivity_config` instead of `context`.

Reply:
These should not change. Will move to config 👍🏻